os/drivers/lwnl/lwnl.c

```diff
@@ -122,6 +121,11 @@ static int lwnl_open(struct file *filep)
 	struct lwnl_upperhalf_s *upper = inode->i_private;
 
 	LWNLDEV_LOCK(upper);
+	/*
+	 * crefs type is uint8 so overflow might happen.
+	 * However there are a few modules which use lwnl.
+	 * So checking overflow is not needed now.
```

Comment:
Do you mean that overflow checking will be done later by some modules which use lwnl?

Reply:
I'll modify it if many modules use lwnl. However only wi-fi manager and iotivity access lwnl now. So I think we don't need to add overflow logic now.